WORKSPACE

```diff
@@ -108,3 +108,21 @@ rbe_autoconfig(
 )
 
 ###############################################################################
+
+###############################################################################
+# Initialize tensorflow workspace.
+load("@org_tensorflow//tensorflow:workspace3.bzl", "tf_workspace3")
```

Comment:
The main perfect doesn't depend on tensorflow -- I think you want to make this change to the integrations/tensorflow workspace file.

Reply:
Well, I didn't know there is intergrations/tensorflow :-) Thanks

Reply:
I found that I made a stupid mistake. I didn't get new submodule `third_party/mlir-hlo`. No wonder, I could not find `@mlir-hlo//:hlo`. Will update this PR later.

Reply:
Submodules accidents happen to us all :)